### ISSUE TITLE
sass inside import css run production terminal display error

### ISSUE THREAD

**Issue by stevieyu** [JeffreyWay/laravel-mix#70]
app.scss
```
@import "~normalize.css/normalize.css";
```

run webpack or dev or hmr display normal
but run production， The following error is displayed

```
  warning  in ./~/css-loader!./~/postcss-loader!./~/resolve-url-loader!./~/sass-loader?sourceMap!./resources/assets/sass/app.scss

  resolve-url-loader cannot operate: CSS error
  /Users/stevie/project/company/code/amii_web/laravel/resources/assets/sass/app.scss:2:474: missing '{'
  at error (/Users/stevie/project/company/code/amii_web/laravel/node_modules/css/lib/parse/index.js:62:15)

 @ ./resources/assets/sass/app.scss 4:14-236
 @ multi app
```

I can not find the reason

**Comment by stevieyu** [comment]
I configured a webpack configuration items, operating normally

https://github.com/stevieyu/vue2-blog

Estimation has nothing to do with the resolve-url-loader

**Comment by stevieyu** [comment]
Thanks released version 0.4.0 to fix this issues
